dotcom-rendering/cypress/support/index.js

```diff
@@ -42,5 +42,12 @@ Cypress.on('uncaught:exception', (err, runnable) => {
 		return false;
 	}
 
+	if (
+		err.message.includes("Cannot read properties of null (reading '__H')")
+	) {
+		console.warn(err);
+		return false;
+	}
```

Comment:
Are we sure that all the banners _actually_ work when this error is ignored? When I was working on this, this error generally indicated that something had gone wrong during the preact rendering process, and that the braze messages were not working.

I could usually verify this by going to the articles in the test and seeing that the banner(s) would not show up, but I didn't manage to figure out the root cause.

Reply:
This was a great call 👍